lib/Pipeline.js

```diff
@@ -82,12 +83,23 @@ Pipeline.processJSON = function(gltf, options) {
  */
 Pipeline.processJSONWithExtras  = function(gltfWithExtras, options) {
     options = defaultValue(options, {});
+
+    addDefaults(gltfWithExtras, options);
+
+    // Print statistics for unoptimized input
+    if (options.stats) {
+        if (options.stats) {
+            console.log("Statistics for " + options.inputPath + "\n------------------\n" +
+                        getStatistics(gltfWithExtras).toString());
+        }
+    }
```

Comment:
`options.stats` appears twice.

Use single quotes.

This would probably look cleaner as two different calls to `console.log`.